toolkit/tools/imager/imager.go

```diff
@@ -577,6 +593,13 @@ func buildImage(mountPointMap, mountPointToFsTypeMap, mountPointToMountArgsMap,
 		}
 	}
 
+	// Run finalize image scripts from within the installroot chroot
```

Comment:
Curious if we should limit this to non-rootfs type images... I vaguely recall having misgivings about allowing this for non-image/disk based install scenarios but I don't recall what they were.

Any thoughts?

Reply:
Only issue I could think of is possible confusion with the post-install scripts field for rootfs only.

Maybe something to do with the bespoke delta update path, which would fall under the category of non-image-based install scenarios?

Reply:
Spoke with @dmcilvaney and we both can't see any issues with the non-image path